run-asp-net-gateway-demo.bat

```diff
@@ -0,0 +1 @@
+build.cmd run-gateway-demo
```

Comment:
minor nitpicks:

- The file extension should be `.cmd` to match the existing ones
- The filename and the target named aren't aligned as with `set-startup-projects`, perhaps rename the file to `run-gateway-demo.cmd`?
- This line should be terminated with a CRLF